src/main/java/TaskList.java

```diff
@@ -0,0 +1,23 @@
+import java.util.ArrayList;
+
+/**
+ * Class representation of TaskList
+ */
+public class TaskList {
```

Comment:
Perhaps you could add more responsibilities to the TaskList class as there are currently only two constructors. An example would be to add retrieveTask(i) and addTask() methods to abstract the adding and removing of Tasks to make it seem like you are dealing with an actual TaskList instead of an ArrayList<Task> object :)